### PR TITLE
tests(page-functions): add test for getNodePath

### PR DESCRIPTION
**Pull request by paulirish**
made this when i was noodling on #10956  

the PR already had shared authorship, so i thought it best to just split this out separate.

ref #10914

### CODE REVIEW

**Comment by paulirish** [comment]
thanks `git reset --soft master && git commit && git push --force`

File: lighthouse-core/lib/page-functions.js [hunk 1]
```diff
@@ -450,6 +450,7 @@ module.exports = {
   computeBenchmarkIndexString: computeBenchmarkIndex.toString(),
   getNodePathString: getNodePath.toString(),
   getNodeSelectorString: getNodeSelector.toString(),
+  getNodePath: getNodePath,
```
Comment (connorjclark) [inline lighthouse-core/lib/page-functions.js:453]:
```suggestion
  getNodePath,
```

File: lighthouse-core/test/lib/page-functions-test.js [hunk 2]
```diff
@@ -126,4 +128,32 @@ describe('Page Functions', () => {
       assert.equal(pageFunctions.getNodeLabel(el), 'div');
     });
   });
+
+  describe('getNodePath', () => {
+    it('Returns basic node path', () => {
```
Comment (connorjclark) [inline lighthouse-core/test/lib/page-functions-test.js:133]:
```suggestion
    it('returns basic node path', () => {
```

File: lighthouse-core/test/lib/page-functions-test.js [hunk 3]
```diff
@@ -126,4 +128,32 @@ describe('Page Functions', () => {
...
+          <span>Sup</span>
+          <img src="#">
+        </section>
+      `;
+      const img = el.querySelector('img');
+      // The img is index 1 of section's children (excluding some whitespace only text nodes)
+      assert.equal(pageFunctions.getNodePath(img), '0,SECTION,1,IMG');
+    });
+
+    it('Returns node path through shadow root', () => {
```
Comment (connorjclark) [inline lighthouse-core/test/lib/page-functions-test.js:146]:
```suggestion
    it('returns node path through shadow root', () => {
```

File: lighthouse-core/test/lib/page-functions-test.js [hunk 4]
```diff
@@ -126,4 +128,32 @@ describe('Page Functions', () => {
...
+    it('Returns basic node path', () => {
+      const el = dom.createElement('div');
+      el.innerHTML = `
+        <section>
+          <span>Sup</span>
+          <img src="#">
+        </section>
+      `;
+      const img = el.querySelector('img');
+      // The img is index 1 of section's children (excluding some whitespace only text nodes)
```
Comment (connorjclark) [inline lighthouse-core/test/lib/page-functions-test.js:142]:
```suggestion
      // The img is index 1 of section's children (excluding some whitespace only text nodes).
```

**Review by adamraine — APPROVED** [review]
LGTM

**Pull request merged**